docs/alma/alma.rst

```diff
@@ -127,17 +127,20 @@ You can query by object name or by circular region:
     >>> from astroquery.alma import Alma
     >>> m83_data = Alma.query_object('M83')
     >>> m83_data.colnames
```

Comment:
this line still gives me a failure (it returns with the original line that you've changed), I would suggest adding `# doctest: +IGNORE_OUTPUT` to the end of this line, so it tests that it runs without issue, but doesn't choke on if it changes. The actual column names then can be tested as part of the test suite, if needed.

Reply:
weird that it's changing order, but I should have been using `IGNORE_OUTPUT` in several places.